### DISCUSSION TITLE
Limited transform rules

### DISCUSSION THREAD

**Discussion by diego-hourly** [micromark/micromark#63] [Q&A] [answered]
I'm trying to create a limited markdown parser that only allows `bold`, `italic`, `strikethrough`, and `code`. Is there a way to tell micromark to disallow rules?

**Comment by wooorm** [comment, marked as answer]
Yup: https://github.com/micromark/micromark/blob/4893cc92dac679c6e53bb21bd32c1e8b536a5adc/test/io/flow/code-indented.js#L78

**Reply by diego-hourly** [reply]
ooh that's awesome! thanks

**Reply by diego-hourly** [reply]
@wooorm is there any docs for `extensions>disable`? can't find which values are allowed

**Reply by wooorm** [reply]
Any construct with a name can be turned off: https://github.com/micromark/micromark/blob/116bfa56b90b6bbc1facddfd0886a7e127a6b03f/packages/micromark/dev/lib/create-tokenizer.js#L453-L458.

Almost all of these have names: https://github.com/micromark/micromark/blob/116bfa56b90b6bbc1facddfd0886a7e127a6b03f/packages/micromark-core-commonmark/dev/index.js.

Docs on extensions are coming soon

**Reply by diego-hourly** [reply]
Awesome! thankyou
